.github/release-drafter.yml

```diff
@@ -0,0 +1,61 @@
+categories:
+  - title: '⚠️  Breaking changes'
+    labels:
+      - 'major'
```

Comment:
```suggestion
      - 'kind/major'
```
I wonder if it should be so

Reply:
I have the same question. But the `kind/*` labels should represent the "kind" or "type" of the change. I'm not 100% sure if this is the case. Maybe should we use `size/major`, 

@kkaempf , what do you think?

Reply:
suggestion: `kind/breaking-change`

Reply:
`size` is like effort estimation, which in reality never really works.
Issues are either "fully defined and actionable" or "unclear, needs refinement".
The former are normal issues, the latter epics. :wink:

Reply:
Done. Now the labels used to set a PR as "major" update can be `kind/breaking-change` or `kind/major`.